cpp/order/Steinhardt.cc

```diff
@@ -0,0 +1,391 @@
+// Copyright (c) 2010-2019 The Regents of the University of Michigan
+// This file is from the freud project, released under the BSD 3-Clause License.
+
+#include "Steinhardt.h"
+
+using namespace std;
+using namespace tbb;
+
+/*! \file Steinhardt.cc
+    \brief Computes variants of Steinhardt order parameters.
+*/
+
+namespace freud { namespace order {
+
+// Calculating Ylm using fsph module
+void Steinhardt::computeYlm(const float theta,
+						    const float phi,
+							std::vector<std::complex<float> > &Ylm)
+    {
+    if (Ylm.size() != 2*m_l+1)
+        {
+        Ylm.resize(2*m_l+1);
+        }
+
+    fsph::PointSPHEvaluator<float> sph_eval(m_l);
+
+    unsigned int j(0);
+    // old definition in compute (theta: 0...pi, phi: 0...2pi)
+    // in fsph, the definition is flipped
```

Comment:
I'd delete this comment, it's referring to a pre-fsph implementation of Steinhardt that must be 3 years old at this point.